packages/patternfly-4/react-core/src/components/Form/FormGroup.js

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormContext } from '../Form/FormContext';
+import { css, getModifier } from '@patternfly/react-styles';
+import styles from '@patternfly/patternfly-next/components/Form/styles.css';
+
+const propTypes = {
+  /** FormGroup field. */
+  children: PropTypes.node,
+  /** Custom class name. */
+  className: PropTypes.string,
+  /** Label text before the field. */
+  label: PropTypes.oneOfType([PropTypes.string, PropTypes.node, PropTypes.func]),
+  /** Sets the FormGroup required. */
+  required: PropTypes.bool,
+  /** Sets the FormGroup invalid. */
+  invalid: PropTypes.bool,
+  /** Helper text after the field. It can be a simple text, a function or an object. */
+  helpertext: PropTypes.oneOfType([PropTypes.string, PropTypes.node, PropTypes.func]),
+  /** Helper text after the field when the field is invalid. It can be a simple text, a function or an object. */
+  helpertextInvalid: PropTypes.oneOfType([PropTypes.string, PropTypes.node, PropTypes.func]),
+  /** ID of the included field. It has to be the same for proper working. */
+  fieldId: PropTypes.string
+};
+
+const defaultProps = {
+  children: null,
+  className: '',
+  label: undefined,
+  required: false,
+  invalid: false,
+  helpertext: undefined,
+  helpertextInvalid: undefined,
+  fieldId: undefined
+};
+
+const renderHelperText = ({ text, classes, fieldId }) => {
+  if (typeof text === 'string') {
+    return (
+      <p className={classes} id={`${String(fieldId)}-helper`} aria-live="polite">
```

Comment:
This assumes that he consumer will use this for the aria-describedby attribute.  You should apply the aria-described by attribute with the same id you use here, when you render the child.

Reply:
Well we should give users some kind of freedom to mess with their forms. I understand that it's kinda overcomplicated to pass multiple times same ID, but given that this is just simplification of combination of label and input we should stick to same logic as when using general `label` and `input` where you have to specify the ID twice as well.